nixos/modules/programs/wayland/wayfire.nix

```
@@ -0,0 +1,55 @@
+{ config, lib, pkgs, ...}:
+
+{
+  meta = {
+    maintainers = with lib.maintainers; [ rewine ];
+  };
+
+  ###### interface
+
+  options.programs.wayfire = {
+    enable = lib.mkEnableOption (lib.mdDoc "Wayfire, a wayland compositor based on wlroots.");
+
+    package = lib.mkPackageOptionMD pkgs "wayfire" { };
+
+    plugins = lib.mkOption {
+      type = lib.types.listOf lib.types.package;
+      default = with pkgs.wayfirePlugins; [ wf-shell ];
+      defaultText = lib.literalExpression "with pkgs.wayfirePlugins; [ wcm wf-shell ]";
```

Given what you said before,

```suggestion
      default = with pkgs.wayfirePlugins; [ wcm wf-shell ];
      defaultText = lib.literalExpression "with pkgs.wayfirePlugins; [ wcm wf-shell ]";
```